tests/epyccel/test_epyccel_sets.py

```diff
@@ -77,42 +77,42 @@ def clear_complex():
     python_result = clear_complex()
     assert python_result == pyccel_result
 
-def test_copy_int(python_only_language):
+def test_copy_int(language):
```

Comment:
Don't you think we should check that the original set is not affected by the `pops` for example? I know it will work now but reflecting their disassociation in the tests will prevent eventual bugs.

Reply:
That is a good idea :+1: I will add this to the tests